geode-core/src/integrationTest/java/org/apache/geode/cache/client/internal/QueueManagerJUnitTest.java

```diff
@@ -238,17 +245,14 @@ public void testWaitForPrimary() {
     manager.start(background);
     manager.getAllConnections().getPrimary().destroy();
 
-    Throwable thrown = catchThrowable(() -> {
-      manager.getAllConnections().getPrimary();
-    });
+    Throwable thrown = catchThrowable(() -> manager.getAllConnections().getPrimary());
     assertThat(thrown).isInstanceOf(NoSubscriptionServersAvailableException.class);
```

Comment:
This can be simplified to:
```
    assertThatThrownBy(() -> manager.getAllConnections().getPrimary())
        .isInstanceOf(NoSubscriptionServersAvailableException.class);
```

Reply:
I didn't want to refactor the whole file for something that is for 1.15

Reply:
Done.